Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/RayTracing/RayTracingSceneSrg.azsli

```diff
@@ -100,6 +110,7 @@ ShaderResourceGroup RayTracingSceneSrg : SRG_RayTracingScene
     #define MESH_BUFFER_FLAG_TANGENT        (1 << 0)
     #define MESH_BUFFER_FLAG_BITANGENT      (1 << 1)
     #define MESH_BUFFER_FLAG_UV             (1 << 2)
+    #define MESH_BUFFER_UINT16_INDEX        (1 << 3)
```

Comment:
- You aren't setting this flag anywhere currently?
- I'd prefer `m_indexFormat` entry instead of a flag, so it's consistent with other format-dependent buffer accesses